hack/generate-operator-bundle.py

```diff
@@ -61,33 +61,6 @@
             'serviceAccountName': 'hive-operator',
         })
 
-# Add our hive-controllers role to the CSV:
-with open('config/rbac/hive_controllers_role.yaml', 'r') as stream:
```

Comment:
Does anything need to be done to clean up the old controller rules for the default service account? Or will that all be handle by OLM?

Reply:
Should be cleaned out by OLM yes.